reconcile/sql_query.py

```diff
@@ -270,9 +270,12 @@ def make_postgres_command(output, sqlqueries, recipient=None):
     command = []
     for query in sqlqueries:
         command.extend([
-            '(time psql',
-            'postgres://$(db.user):$(db.password)@'
-            '$(db.host):$(db.port)/$(db.name)',
+            '(PGPASSWORD="$(db.password)"',
+            'time psql',
+            '--host=$(db.host)',
+            '--port=$(db.port)',
+            '--username=$(db.user)',
+            '--dbname=$(db.name)'
             f'--command "{query}")',
```

Comment:
@maorfr 
You have a risk here with "{query}" as well.

If the query itself contains double-quote chars, you could encounter other bash errors. There should be a more secure way to do this.

Ex:
```sql
select 1 as "cool_column"
  from my_table as "eek";
```

Reply:
thanks for raising this! this is not an issue, as the query returns escaped from the graphql server. example:
```
      {
        "name": "2021-01-19-compliance-prod-check-profile-policies",
        "query": "SELECT \"version\" FROM \"schema_migrations\" ORDER BY \"version\" DESC LIMIT 10;"
      }
```

Reply:
thanks for raising this! this is not an issue, as the query returns escaped from the graphql server. example:
```
      {
        "query": "SELECT \"version\" FROM \"schema_migrations\" ORDER BY \"version\" DESC LIMIT 10;"
      }
```